sql/server.go

```diff
@@ -37,13 +37,14 @@ import (
 )
 
 var (
-	allowedEncodings     = []util.EncodingType{util.JSONEncoding, util.ProtoEncoding}
-	errNoDatabase        = errors.New("no database specified")
-	errNoTable           = errors.New("no table specified")
-	errEmptyDatabaseName = errors.New("empty database name")
-	errEmptyTableName    = errors.New("empty table name")
-	errEmptyIndexName    = errors.New("empty index name")
-	errEmptyColumnName   = errors.New("empty column name")
+	allowedEncodings      = []util.EncodingType{util.JSONEncoding, util.ProtoEncoding}
+	errNoDatabase         = errors.New("no database specified")
+	errNoTable            = errors.New("no table specified")
+	errEmptyDatabaseName  = errors.New("empty database name")
+	errEmptyTableName     = errors.New("empty table name")
+	errEmptyIndexName     = errors.New("empty index name")
+	errEmptyColumnName    = errors.New("empty column name")
+	errBeginDuringOpenTxn = errors.New("begin transaction during an open transaction")
```

Comment:
perhaps this deserves a comment: in postgres this results in a warning and is a no-op